### ISSUE TITLE
[pt-br] Localize docs/tasks/access-application-cluster/web-ui-dashboard/

### ISSUE THREAD

**Issue by nataliagranato** [kubernetes/website#46782]
Localize docs/tasks/access-application-cluster/web-ui-dashboard/ to pt-br.

**Comment by edsoncelio** [comment]
Hi @nataliagranato!
I just wanted to remind you about this issue - are you still working on it?
If you need any help, don't hesitate to let us know.

**Comment by nataliagranato** [comment]
> Hi @nataliagranato! I just wanted to remind you about this issue - are you still working on it? If you need any help, don't hesitate to let us know.

@edsoncelio Yes, I am almost finished. I had some health problems, but I haven't forgotten.

**Comment by nataliagranato** [comment]
@edsoncelio The pull request with the translation has been opened at https://github.com/kubernetes/website/pull/48562

**Comment by stormqueen1990** [comment]
This work was completed via #48563
/close
